test/meta/Makefile

```diff
@@ -0,0 +1,27 @@
+# SpiderMonkey shell
+JSSHELL=~/m-i/js/src/build-debug/dist/bin/js -e 'const WITH_SHARED_MEMORY=false;' -f common.js
```

Comment:
We might want to add a way to use `d8` or `node` here too, but this is fine for now.